### ISSUE TITLE
How to pre-process the AffectNet dataset?

### ISSUE THREAD

**Issue by hanguojiang0821** [siqueira-hc/Efficient-Facial-Feature-Learning-with-Wide-Ensemble-based-Convolutional-Neural-Networks#14]
when I use "pre_process_affect_net" to process the images, I only got the Validation File， Training_Labeled File and Training_Unlabeled File are not available. Is my  base_path_to_images, base_path_to_annotations setting wrong? How can I handle this? Looking forward to your reply!

**Comment by siqueira-hc** [comment]
Dear hanguojiang0821,

Thank you very much for your message.

You can now choose the set to be processed by setting the argument set_index.

set_index = 0 process the automatically annotated images.
set_index = 1 process the manually annotated images: training set.
set_index = 2 process the manually annotated images: validation set.

Best,
Henrique Siqueira

**Comment by hanguojiang0821** [comment]
Thank you very much for your reply, I found this question later and successfully run the script.
Thanks again for your answers！
